test/v1/controllers/CreatePeriodSummaryControllerSpec.scala

```diff
@@ -251,7 +251,7 @@ class CreatePeriodSummaryControllerSpec
           }
         }
 
-        val input = Seq(
+        val errors = Seq(
```

Comment:
Looking at other controllers, there are Tys specific test. Maybe we are missing that?

Reply:
Hey Qing, do you have an example of that? We're not quite sure what you mean by a specific test. I don't think there's much use in refactoring the errors test, as they will be removed when we implement Matt's MTD error changes. I don't see the benefit in having a TYS specific happy path test, as all the TYS code changes are within the connector & service, which are stubbed out in the controller spec.

Reply:
I don't see the benefit as well to be honest for this case.
Its just that for some controller has it.  e.g CreateAmendUkDividendsAnnualSummaryControllerISpec.scala in individuals-income-received-api

Reply:
happy to ignore it